app/Jobs/Emails/AbstractSummitEmailJob.php

```diff
@@ -75,8 +76,22 @@ public function __construct
         $payload[IMailTemplatesConstants::registration_link] = $summit->getRegistrationLink();
         $payload[IMailTemplatesConstants::virtual_event_site_link] = $summit->getVirtualSiteUrl();
 
+        $payload = array_merge($payload, self::getMarketingVariables($summit));
+        parent::__construct($payload, $template_identifier, $to_email, $subject, $cc_email, $bcc_email);
+    }
+
+    private static function getMarketingVariables(Summit $summit) {
         $marketing_api = App::make(IMarketingApi::class);
         $marketing_vars = $marketing_api->getConfigValues($summit->getId(), 'EMAIL_TEMPLATE_');
+
+        $default_email_template_vars = Config::get('marketing.default_email_template_vars');
+        foreach ($default_email_template_vars as $key => $value) {
+            $key = strtoupper($key);
+            if (!array_key_exists($key, $marketing_vars)) {
```

Comment:
if not just exists
u need to check that also has something, if not set
then set the default one